lib/collection/property-base.js

```diff
@@ -35,6 +36,38 @@ PropertyBase = function PropertyBase (options) {
     _.keys(meta).length && (this._ = meta);
 };
 
+_.extend(PropertyBase.prototype, {
+    toJSON: function () {
+        return _.reduce(this, function (accumulator, value, key) {
+            if (value === undefined) { // true/false/null need to be preserved.
+                return accumulator;
+            }
+
+            // Handle plurality of PropertyLists in the SDK vs the exported JSON.
+            // Basically, removes the trailing "s" from key if the value is a property list.
+            if (PropertyList.isPropertyList(value) && _.endsWith(key, 's')) {
+                key = key.slice(0, -1);
+            }
+
+            // Handle 'Property's
+            if (_.isFunction(value.toJSON)) {
+                accumulator[key] = value.toJSON();
+                return accumulator;
+            }
+
+            // Handle Strings
+            if (_.isString(value)) {
+                accumulator[key] = value;
+                return accumulator;
+            }
+
+            // Everything else
+            accumulator[key] = _.clone(value);
```

Comment:
Clone is shallow. Should it not be cloneDeep? Think objects. But problem with clone deep is that it will also start cloning properties encountered as grandchild. You'll need to implement your own clone deep that respects toJSON.